src/Parser/DeathKnight/Blood/Modules/Features/CrimsonScourge.js

```diff
@@ -50,34 +50,12 @@ class CrimsonScourge extends Analyzer {
     return this.wastedCrimsonScourgeProcs / this.crimsonScourgeProcsCounter;
   }
 
-  get suggestionThresholds() {
-    return {
-      actual: this.wastedCrimsonScourgeProcsPercent,
-      isGreaterThan: {
-        minor: 0.05,
-        average: 0.1,
-        major: 0.15,
-      },
-      style: 'percentage',
-      text: 'You had unspent Crimson Scourge procs. Make sure you always use them.',
-    };
-  }
-
-  suggestions(when) {
-    when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
```

Comment:
Why remove this suggestion?

Reply:
Don't need it now that DnD has cast efficiency. Although saying that I should probably enable it if Rapid Decomposition talent isn't used and disable the DnD efficiency in that case.